.gitignore

```diff
@@ -27,6 +27,7 @@ lcov_cobertura.py
 
 # state file created during tests
 ferox-*.state
+feroxbuster-*.profraw
```

Comment:
with the change to LLVM_PROFILE_FILE, this shouldn't be necessary since it's covered by the `target/` line